client/src/modules/inventory/list/templates/action.cell.html

```diff
@@ -14,5 +14,13 @@
           <i class="fa fa-edit"></i> <span translate>FORM.BUTTONS.EDIT</span>
       </a>
     </li>
+
+    <li class="divider"></li>
+    <li>
+      <a data-method="invoice" ui-sref="invoiceRegistry({ filters : { key : 'inventoryLabel', value : row.entity.label }})" href>
```

Comment:
If you use the `uuid`, the query will be more efficient.  To show a nice label instead of a giant UUID, try passing in a display value like this:
```html
ui-sref="invoiceRegistry({ filters : { key : 'inventory_uuid', value : row.entity.uuid, displayValue : row.entity.label }})"
```
Then, in the invoice registry, you can bind the `displayValue` to the filter and it should filter correctly.